atum/src/main/scala/za/co/absa/atum/AtumImplicits.scala

```diff
@@ -260,6 +261,34 @@ trait AtumImplicitsBase {
       dataset
     }
 
+    /**
+     * The method returns ControlMeasure object from the Atum context
+     * @return - ControlMeasure object
+     */
+    def getControlMeasure: ControlMeasure = {
```

Comment:
Remember we are doing this as a DF implicit. So maybe the method names should be more specific, like `getAtumControlMeasure` and `getAtumAdditionalInfo`. What do you think?

Reply:
Agree on the second. Not sure if needed for the first, control measure is a rather specific feature, partly a synonym of Atum.

Reply:
Decided to keep getControlMeasure name because other methods in AtumImplicits also doesn't contain the word Atum.
And realized that I can call atum.controlFrameworkState.getControlMeasure instead of atum.controlFrameworkState.accumulator.getControlMeasure.